packages/account/src/Styles/account.scss

```
@@ -2553,9 +2553,18 @@ $MIN_HEIGHT_FLOATING: calc(
         }
 
         &__input {
-            margin: 16px 0 0;
+            margin: 8px 0 0;
```

I think it's better to use `rem` here.

```suggestion
            margin: 0.8rem 0 0;
```